### PR TITLE
Tag admin improvements

### PR DESCRIPTION
**Pull request by demiankatz**
This PR simplifies some of the tag administration code and fixes minor bugs. It is inspired by work/discussion on #2399.

### CODE REVIEW

File: module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php [hunk 1]
```diff
@@ -197,7 +193,7 @@ protected function getConfirmDeleteMessages($count)
         // Default all messages to "All"; we'll make them more specific as needed:
         $userMsg = $tagMsg = $resourceMsg = 'All';
```
Comment (xmorave2) [inline module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php]: Could you translate the 'All' string here? I know that not all string in admin module are translated, but when you do refactoring here it should not harm ta add this minor change.

**Review by xmorave2 — CHANGES_REQUESTED** [review]
Looks good to me, just one minor suggestion.

File: module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php (hunk 1 above)
Comment (demiankatz) [reply module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php]: Thanks for pointing this out -- should be fixed now.

**Review by xmorave2 — APPROVED** [review]
Thanks @demiankatz.

**Pull request merged**